x-pack/plugins/ml/public/application/data_frame_analytics/pages/job_map/page.tsx

```diff
@@ -120,7 +120,7 @@ export const Page: FC = () => {
         </MlPageHeader>
       ) : null}
       {jobId !== undefined ? (
-        <MlPageHeader key={`${jobId}-id`}>
+        <MlPageHeader>
```

Comment:
If you select from the models tab of the selector, you end up with this:

![image](https://user-images.githubusercontent.com/7405507/173867442-df37dd06-ad11-49d0-bc71-bfb04e5a16f7.png)

Reply:
https://github.com/elastic/kibana/pull/134466/commits/8ff51b7ff555fd6b3fd4ed09fcb76b73b94a8ee1 removed another `key`,  this is the last one